playground-test/components/feature.jsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
```

Comment:
I'm probably missing something.
What's the difference between `playground-test` and `playground`?

Also, I see the components appearing twice in `playground-test`.
We have
`playground-test/components/feature.jsx`
and also
`playground-test/src/components/Feature.js`
Do we need both?

Reply:
That was a conscious decision. Playground test is not meant to change (unless you change the tests) and meant for testing. While playground is meant to mess around

Reply:
Hey @alexjoverm 
What does it mean to "mess around" with `playground`?
On the other hand, how is `playground-test` being used in the tests?

Besides that, is there any reason why we need duplicate components in `playground-test`? We have a `playground-test/components/`and a `playground-test/src/components/` folders, and the components inside of them look kind of the same.

Reply:
True, not necessary to duplicate those components (might be leftovers of copy/paste).

About why need both... Maybe in next meeting?

Reply:
Nice! 😄

Reply:
Het @alexjoverm and @fgiuliani 
Good catch again! I'll remove `playground-test/components` and `playground/components` as these files are duplications by mistake👍

Include me in the next meeting about that, Alex🙏